llama-index-integrations/readers/llama-index-readers-minio/llama_index/readers/minio/minio_client/base.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         file_metadata: Optional[Callable[[str], Dict]] = None,
         minio_endpoint: Optional[str] = None,
         minio_secure: bool = False,
+        minio_cert_check: bool = False,
```

Comment:
Shouldn't this be `True` by default as a security best practice?

Reply:
You're right. I'll fix that ASAP